embedding/esp8266/.gitignore

```diff
@@ -0,0 +1,4 @@
+user/esp8266_js.h
```

Comment:
yes, to ignore build time generated files and it be dependent on targets. can I ask if you have some other things in mind?